tensorflow/core/util/mkl_util.h

```diff
@@ -83,7 +59,6 @@ namespace tensorflow {
 // MKL operation, and did not go through a conversion to a standard
 // Tensorflow tensor.
 
-// For use with MKL ML, has been deprecated
 typedef enum { W = 0, H = 1, C = 2, N = 3 } MklDims;
```

Comment:
If this is only used in MKL-ML (according to the comment above), should it be removed as well?

Reply:
Good point, let me double-check. Thanks!

Reply:
Done by removing the enum and related deprecated method GetMklTensorDimIndex.